packages/core/src/components/forms/numericInput.tsx

```diff
@@ -47,6 +47,13 @@ export interface INumericInputProps extends IIntentProps, IProps {
     disabled?: boolean;
 
     /**
+     * If set to `true`, the input will display with larger styling.
+     * This is equivalent to setting `pt-large` via className.
```

Comment:
but it sets `.pt-large` class on multiple elements, so they're not exactly equivalent. worth noting the difference here?